INSTALL.md

```diff
@@ -2,39 +2,41 @@
 
 #### Download
 
-Download using the [GitHub .zip download](https://github.com/dracula/windows-terminal/archive/master.zip) option
+Download using the [GitHub .zip download](https://github.com/dracula/windows-terminal/archive/master.zip) option.
 
 #### Install
 
 Start Windows Terminal and click on the down arrow symbol `˅` from menu bar. This will open a drop down menu from which select Settings option. Alternatively use `Ctrl + ,` to open Settings directly.
 
-In the `profile.json` settings file for Windows Terminal, find the `schemes` section and paste the content of `dracula.json`.
+In the `settings.json` settings file for Windows Terminal, find the `schemes` section and paste the content of `dracula.json`.
 
 Example:
 
 ```json
 "schemes": [
-    {
-        "name" : "Dracula",
-        "background" : "#272935",
-        "black" : "#21222C",
-        "blue" : "#BD93F9",
-        "cyan" : "#8BE9FD",
-        "foreground" : "#F8F8F2",
-        "green" : "#50FA7B",
-        "purple" : "#FF79C6",
-        "red" : "#FF5555",
-        "white" : "#F8F8F2",
-        "yellow" : "#F1FA8C",
-        "brightBlack" : "#6272A4",
-        "brightBlue" : "#D6ACFF",
-        "brightCyan" : "#A4FFFF",
-        "brightGreen" : "#69FF94",
-        "brightPurple" : "#FF92DF",
-        "brightRed" : "#FF6E6E",
-        "brightWhite" : "#FFFFFF",
-        "brightYellow" : "#FFFFA5"
-    }
+  {
+    "name": "Dracula",
+    "cursorColor": "#F8F8F2",
+    "selectionBackground": "#44475A",
+    "background": "#282A36",
+    "foreground": "#F8F8F2",
+    "black": "#21222C",
+    "red": "#FF5555",
+    "green": "#50FA7B",
+    "yellow": "#F1FA8C",
+    "blue": "#BD93F9",
+    "purple": "#FF79C6",
+    "cyan": "#8BE9FD",
+    "white": "#F8F8F2",
+    "brightBlack": "#6272A4",
+    "brightRed": "#FF6E6E",
+    "brightGreen": "#69FF94",
+    "brightYellow": "#FFFFA5",
+    "brightBlue": "#D6ACFF",
+    "brightPurple": "#FF92DF",
+    "brightCyan": "#A4FFFF",
+    "brightWhite": "#FFFFFF"
+  }
```

Comment:
Need to keep the sorting consistent so the diff highlights what has been changed consistently.

```
name
a...z - normal options
brightA...brightZ - brightColors
```

Reply:
I originally ordered it the way I did to be consistent with the [Dracula spec](https://spec.draculatheme.com/#sec-ANSI) and the ordering of the [ANSI color codes](https://en.wikipedia.org/wiki/ANSI_escape_code#3/4_bit).

Looking at the [Windows Terminal docs on color schemes](https://docs.microsoft.com/en-us/windows/terminal/customize-settings/color-schemes#creating-your-own-color-scheme), I realize those are closer to the original ordering of this repo. The ANSI colors are split into 2 groups and sorted alphabetically within them, but `cursorColor`, `selectionBackground`, `background`, and `foreground` are split out separately at the top. I figure it makes the most sense to be consistent with the Window Terminal docs, so I changed it to this ordering (also changed the indentation back to 4 spaces), is that alright?

The diffs are still not great, but that's because I took out the space between each property and `:`. I figure consistency with the Windows Terminal docs and standard JSON formatting should be the more important than a nice diff, but I'm fine with changing it to make the diff look nicer.

Reply:
Works for me. Thanks.